accessibility-checker-engine/help-v4/en-US/style_before_after_review.html

```diff
@@ -68,14 +70,14 @@ <h3 id="ruleMessage"></h3>
 ### About this requirement
 
 * [IBM 1.3.1 Info and Relationships](https://www.ibm.com/able/requirements/requirements/#1_3_1)
-* [WCAG failure F87: Due to inserting non-decorative content by using ::before and ::after pseudo-elements and the 'content' property in CSS](https://www.w3.org/WAI/WCAG22/Techniques/failures/F87.html)
+* [Failure F87: CSS ::before and ::after pseudo-elements for content](https://www.w3.org/WAI/WCAG22/Techniques/failures/F87.html)
 
 ### Who does this affect?
 
  * People with visual impairment using color contrast enhancement
  * People who use alternate backgrounds to increase legibility
  * People who personalize their display colors for easier reading
- * People using text only, monochrome or braille displays
+ * People using text-only, monochrome, or Braille displays
```

Comment:
Lowercase was correct
```suggestion
 * People using text-only, monochrome, or braille displays
```